src/main/java/carpet/logging/Logger.java

```diff
@@ -74,11 +74,7 @@ public String getDefault()
     }
     public String [] getOptions()
     {
-        if (options == null)
-        {
-            return new String[0];
-        }
-        return options;
+        return options == null ? new String[0] : options;
```

Comment:
Could also set the array to an empty one at constructor if it's null instead of checking every time tbh. Not really important.